EZSwiftExtensionsTests/UILabelTests.swift

```diff
@@ -7,10 +7,45 @@
 //
 
 import XCTest
-
+@testable import EZSwiftExtensions
 class UILabelTests: XCTestCase {
 
+    func testInit() {
+        
+        let label = UILabel(x: 0, y: 0, w: 200, h: 50)
+        let expected = UILabel(frame: CGRect(x: 0, y: 0, width: 200, height: 50))
+        let label2 = UILabel(x: 0, y: 0, w: 200, h: 50, fontSize: 20)
+        
+        XCTAssertEqual(label.frame, expected.frame)
+        XCTAssertEqual(label2.font.pointSize, 20)
+    }
     
+    func testText() {
+        
+        let label = UILabel(frame: CGRect(x: 0, y: 0, width: 200, height: 50))
+        label.setText("EZSwiftExtensions✅", animated: false, duration: nil)
+        XCTAssertEqual(label.text, "EZSwiftExtensions✅")
+        
+        label.text = ""
+        label.setText("EZSwiftExtensions🚀", animated: true, duration: 5)
+        XCTAssertEqual(label.text, "EZSwiftExtensions🚀")
+        
+        label.text = ""
+        label.setText("EZSwiftExtensions❤️", animated: true, duration: nil)
+        XCTAssertEqual(label.text, "EZSwiftExtensions❤️")
+    }
 
+    var waitExpectation: XCTestExpectation?
```

Comment:
@goktugyil this is for asynchronous operations, test are not exited immediately, they are waiting for expectation